### DISCUSSION TITLE
How to use predictor.pkl to predict?

### DISCUSSION THREAD

**Discussion by ghost** [autogluon/autogluon#4751] [Q&A]
predictor = joblib.load(predictor.pkl)
predictor.predict(ts)

crash：'NoneType' object has no attribute 'predict'

**Comment by ghost** [comment]
i also load DirectTabular/model.pkl
but fail with 'NoneType' object has no attribute 'predict'

**Comment by LennartPurucker** [comment]
Heyho, I think you are looking for the `.load()` function of the predictor class:
* For tabular: https://auto.gluon.ai/stable/api/autogluon.tabular.TabularPredictor.load.html
* For TimeSeries: https://auto.gluon.ai/stable/api/autogluon.timeseries.TimeSeriesPredictor.load.html
* For MultiModal: https://auto.gluon.ai/stable/api/autogluon.multimodal.MultiModalPredictor.load.html
